app/src/main/res/values/strings.xml

```diff
@@ -369,10 +369,10 @@
     <string name="version_details">Version details</string>
     <string name="download_now">Download Now</string>
     <string name="automatically_check_for_updates">Automatically check for updates</string>
-    <string name="here_you_can_save_the_settings_to_the_external_storage_sdcard">Here you can save the settings to the external storage (SD card).\n\nOnce saved to the external storage it is potentially possible for any app to read the settings which might contain sensitive information.\n\nYou are advised to delete the settings once you have imported them again if you are concerned about this.</string>
-    <string name="save_all_settings_to_sdcard">Save all Settings to SD card</string>
-    <string name="load_all_settings_from_sdcard">Load all Settings from SD card</string>
-    <string name="delete_any_settings_on_sdcard">Delete any settings on SD card</string>
+    <string name="here_you_can_save_the_settings_to_the_external_storage_sdcard">Here, you can save the settings to the internal storage (or SD card).\n\nOnce saved to a file, it is potentially possible for any app to read the settings which might contain sensitive information.\n\nYou are advised to delete the settings once you have imported them again if you are concerned about this.</string>
```

Comment:
I would remove the brackets around "or SD card" because "or" is a grammatical conjunction and I don't see any meaning of the brackets, here.

Reply:
Maybe, we can shortened it to "Here, you can save the settings.". Alternatively, we can think about improving consistency and replace all Load/Save staff with Import/Export because it is below "Import/Export features".

Reply:
Thanks for looking at this.

After I submitted this, I wasn't happy with leaving or SD card.  When I use this to save settings, it is always saved to the Download folder.  I don't know how you can save to the SD card even if you want to.  So, unless there is any objection, I would like to remove or SD card and the brackets.  But, I'd rather not just remove the brackets.

I wouldn't change to "here, you can save the settings".  The fact that the settings are saved to the internal storage, which is outside the boundary of xDrip, is very important for the user.  That's what this paragraph is getting at by the warning that the user should be aware of the possibility of revealing important information if someone copies the saved file without their knowledge.